login-workflow/src/components/RegistrationWorkflow/RegistrationWorkflow.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { useEffect, useState } from 'react';
+import React, { cloneElement, useEffect, useState } from 'react';
+import { useNavigate } from 'react-router';
```

Comment:
Our workflow package is router independent...we can't use react-router inside it.

Reply:
removed